### ISSUE TITLE
Running the contributed command: 'language-julia.changeCurrentEnvironment' failed.

### ISSUE THREAD

**Issue by Jarartur** [julia-vscode/julia-vscode#2164]
I installed Julia 1.6.1 with `jill install` in my python3 dev space (Docker container). When i try to use vs code extension it gets stuck on `Starting Julia Language Server` and shows above error.

Julia is in PATH and i can use it from a terminal. I also tried to manually set `julia.environmentPath` but no luck.

Also, the first time it didn't install `LanguageServer` so I added it manually from Julia.

**Version Info:**
VS Code: 1.56.2
Julia: 1.6.1
Extension: 1.1.40
Container OS: Debian 10
Docker: 20.10.6

**Comment by davidanthoff** [comment]
Can you start a Julia REPL inside VS Code?

**Comment by Jarartur** [comment]
Yes, I can use the REPL and evaluate manually but language hints/docs panel/inline evaluation doesn't work.
I start the REPL by just calling julia in my integrated terminal. I can use it normally, install packages, etc. It just doesn't seem to connect with the extension.

**Comment by davidanthoff** [comment]
Does the command `Julia: Start REPL` work, rather than starting Julia in a shell?

**Comment by Jarartur** [comment]
Unfortunately, no. The whole extension is stuck in a loading phase trying to detect Julia installation.

**Comment by davidanthoff** [comment]
Closing as outdated: the language server now requires Julia 1.11 or newer (it refuses to start below that), so the configuration in this report is no longer supported. Juliaup, environment discovery and language server startup have all been reworked since as well.

If you hit something similar on a current setup, please open a new issue with the details and we will take a look.
